docker-compose.yml

```diff
@@ -7,9 +7,7 @@ services:
       - MYSQL_USER=remo
       - MYSQL_PASSWORD=remo
       - MYSQL_ROOT_PASSWORD=root
-    entrypoint:
-      - docker-entrypoint.sh
-      - --max_allowed_packet=2M
+    command: mysqld --max_allowed_packet=64M
```

Comment:
I don't remember right now, the best way is to replicate with http://remo.readthedocs.io/en/latest/installation-docker.html and see what happen.
I know only that in my case I get that error of missing file (in the repo this file doesn't exist)